frontend/taipy-gui/src/components/Taipy/Alert.tsx

```diff
@@ -70,7 +70,7 @@ const Alert = ({ alerts }: AlertProps) => {
                 });
                 alert.system && new Notification(document.title || "Taipy", { body: alert.message, icon: faviconUrl });
             }
-            dispatch(createDeleteAlertAction());
```

Comment:
- on creation (type != ""), you need to pass the id as a key to `enqueueSnackbar` options
- on deletion, you need to use that id as an argument to  `closeSnackbar`
- I guess we don't need the lastKey ref anymore